mne/tests/test_evoked.py

```diff
@@ -452,3 +452,33 @@ def test_array_epochs():
     types = ['eeg'] * 19
     info = create_info(ch_names, sfreq, types)
     assert_raises(ValueError, EvokedArray, data1, info, tmin=-0.01)
+
+
+def test_add_channels():
+    """Test evoked splitting / re-appending channel types
+    """
+    evoked = read_evokeds(fname, condition=0)
+    evoked.info['buffer_size_sec'] = None
```

Comment:
ah ok - so then are we good on this PR?

On Thu, Jul 23, 2015 at 5:39 PM, Alexandre Gramfort <
notifications@github.com> wrote:

> In mne/tests/test_evoked.py
> <https://github.com/mne-tools/mne-python/pull/2302#discussion_r35335545>:
>
> > @@ -452,3 +452,33 @@ def test_array_epochs():
> >      types = ['eeg'] * 19
> >      info = create_info(ch_names, sfreq, types)
> >      assert_raises(ValueError, EvokedArray, data1, info, tmin=-0.01)
> > +
> > +
> > +def test_add_channels():
> > +    """Test evoked splitting / re-appending channel types
> > +    """
> > +    evoked = read_evokeds(fname, condition=0)
> > +    evoked.info['buffer_size_sec'] = None
>
> that's a bug in the evoked IO code that needs to be fixed in another PR...
>
> grrrr
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/2302/files#r35335545>.
>